doc/developers.rst

```diff
@@ -537,12 +569,17 @@ the case of builtin's there is kernel metadata but it is part of
 PSyclone and is specified in
 `src/psyclone/dynamo0p3_builtins_mod.f90`.
 
+For inter-grid kernels, it is the coarse mesh that provides the iteration
+space. (The kernel is passed a list of the cells in the fine mesh that are
+associated with the current coarse cell.)
+
 Cell iterators: Continuous
 --------------------------
 
 When a kernel is written to iterate over cells and modify a continuous
 field, PSyclone always computes dofs on owned cells and redundantly
-computes dofs in the level-1 halo. Users can apply a redundant
+computes dofs in the level-1 halo (or level-2 if the field is on the
+fine mesh of an inter-grid kernel). Users can apply a redundant
```

Comment:
I'm confused - I thought level-2 *was* depth 2? Anyway, I've updated the text...

Reply:
Referencing the multigrid section does the job, thanks. The issue with level-2 wasn't the name, it was that the way it was written meant that it could have been read as just the level 2-halo (and not the level-1 halo as well as the level-2 halo). I would have been happy with `or to level-2` as much as `or to depth 2`